app/controllers/analyses_controller.rb

```diff
@@ -0,0 +1,78 @@
+class AnalysesController < ApplicationController
+  before_filter :authenticate_user!, except: :new
```

Comment:
Fixed. It was intentional only to show the first introductory view, not the form (just like it is with submissions).